lib/src/op_store.rs

```diff
@@ -417,6 +418,12 @@ pub enum OpStoreError {
 
 pub type OpStoreResult<T> = Result<T, OpStoreError>;
 
+impl From<OpHeadStoreError> for OpStoreError {
+    fn from(err: OpHeadStoreError) -> Self {
+        err.into()
+    }
+}
+
```

Comment:
Perhaps, `op_walk::get_current_head_ops()` can return `OpsetEvaluationError`. It's not actually an "op set" thing, but these functions belong to the same API layer.

Reply:
We ended up taking yuja's suggestion for using OpsetEvaluationError since we wanted to reuse that from the beginning, but were unsure whether we should or create a new error